test/exception_notifier/sidekiq_test.rb

```diff
@@ -24,10 +24,12 @@ class SidekiqTest < ActiveSupport::TestCase
     message = {}
     exception = RuntimeError.new
 
-    ExceptionNotifier.expects(:notify_exception).with(
-      exception,
-      data: {sidekiq: message}
-    )
+    ExceptionNotifier.expects(:notify_exception).with do |exc, options|
```

Comment:
I think here would be a good place to modify the assertion based on `::Sidekiq::VERSION < "7.1.5"`

Reply:
I can be convinced. It's what I had originally.